gap/measure_contention.g

```diff
@@ -26,55 +23,80 @@ GAUSS_prepare := function(nrAvailableThreads)
     fi;
 end;
 
-GAUSS_compute := function(A, q, numberChops)
-    return GAUSS_GET_REAL_TIME_OF_FUNCTION_CALL(Chief,
+GAUSS_compute := function(A, q, numberChops, showOutput)
+    if (showOutput) then
+        Print("Starting the parallel execution.\n");
+    fi;
+    res := GAUSS_GET_REAL_TIME_OF_FUNCTION_CALL(Chief,
```

Comment:
Where did you declare res?

Reply:
Good catch. Apparently I overlooked the syntax error message in the output of `LoadPackage`.

Reply:
I amended the commit.